src/Metadata/MapperMetadata.php

```diff
@@ -67,7 +68,7 @@ public function getHash(): string
             }
         }
 
-        $hash .= AutoMapper::VERSION;
+        $hash .= InstalledVersions::getVersion('jolicode/automapper');
```

Comment:
I think you should add `composer-runtime-api` to the `require` section of your `composer.json` when using this.

Reply:
Thanks didn't know that